src/applications/gi/components/search/KeywordSearch.jsx

```diff
@@ -82,22 +65,42 @@ export class KeywordSearch extends React.Component {
           htmlFor="institution-search">
           {this.props.label}
         </label>
-        <Autosuggest
-          getSuggestionValue={this.clickedSuggestionValue}
-          highlightFirstSuggestion
-          onSuggestionsClearRequested={this.props.onClearAutocompleteSuggestions}
-          onSuggestionSelected={this.handleSuggestionSelected}
-          onSuggestionsFetchRequested={this.handleFetchSuggestion}
-          renderSuggestion={this.renderSuggestion}
-          shouldRenderSuggestions={this.shouldRenderSuggestions}
-          suggestions={suggestions}
-          inputProps={{
-            value: searchTerm,
-            onChange: this.handleChange,
-            onKeyUp: this.handleKeyUp,
-            'aria-labelledby': 'institution-search-label',
-            onBlur: this.handleInputBlur,
-          }}/>
+        <Downshift onChange={this.handleChange} onSelect={(item) => this.handleSuggestionSelected(item)}>
+          {({
+            getInputProps,
+            getItemProps,
+            isOpen,
+            inputValue,
+            highlightedIndex
+          }) => (
+            <div>
+              <input {...getInputProps({
+                value: searchTerm || inputValue,
+                onChange: this.handleChange,
+                onKeyUp: this.handleKeyUp,
+                'aria-labelledby': 'institution-search-label',
+                onBlur: this.handleInputBlur
+              })}/>
+              {isOpen && (<div className="suggestions-list">
+                {suggestions.map((item, index) => (
+                  <div
+                    key={index}
+                    className="suggestion"
+                    {...getItemProps({
+                      item: item.label,
+                      style: {
```

Comment:
Ideally, I think we'd toggle a class on and off here. We use a module called `classnames` to make that easier.

Reply:
k